fix: LEAP-2063: LEAP-2062: No more annotations message is displayed in Review Stream when Interactive View All FF is turned ON

nass600 opened the pull request:
Having the `Interactive View All` flag enabled is generating extra renders in the tree with some undefined data which is throwing uncontrolled errors hence the app is behaving erratically given we are accessing a detached state tree's settings.

It should fix:
- https://humansignal.atlassian.net/browse/LEAP-2063
- https://humansignal.atlassian.net/browse/LEAP-2062

<!--

This description MUST be filled out for a PR to receive a review. Its primary purposes are:

 - to enable your reviewer to review your code easily, and
 - to convince your reviewer that your code works as intended.

Some pointers to think about when filling out your PR description:
 - Reason for change: Description of problem and solution
 - Screenshots: All visible changes should include screenshots.
 - Rollout strategy: How will this code be rolled out? Feature flags / env var / other
 - Testing: Description of how this is being verified
 - Risks: Are there any known risks associated with this change, eg to security or performance?
 - Reviewer notes: Any info to help reviewers approve the PR
 - General notes: Any info to help onlookers understand the code, or callouts to significant portions.

You may use AI tools such as Copilot Actions to assist with writing your PR description (see https://docs.github.com/en/copilot/using-github-copilot/using-github-copilot-for-pull-requests/creating-a-pull-request-summary-with-github-copilot); however, an AI summary isn't enough by itself. You'll need to provide your reviewer with strong evidence that your code works as intended, which requires actually running the code and showing that it works.

-->

nass600 committed:
fix: LEAP-2063: No more annotations message is displayed in Review Stream when Interactive View All FF is turned ON

nass600 committed:
reduced the change scope

hlomzik commented on web/libs/editor/src/mixins/AnnotationMixin.js:
```diff
@@ -23,18 +26,9 @@ export const AnnotationMixin = types.model("AnnotationMixin", {}).views((self) =
-      }
-
-      // return connected annotation, not the globally selected one
-      return Types.getParentOfTypeString(self, "Annotation");
```
That is the crucial part of feature flagged code brand, we should not remove it just like this.

nass600 committed:
fix amended

nass600 committed:
removed blank line

AndrejOros commented:
/fm sync
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/14831533545)

AndrejOros committed:
Merge branch 'develop' into 'fb-leap-2063/no-more-annotations'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/14831533545

AndrejOros committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/14831533545
